README.md

```diff
@@ -1 +1,87 @@
-# Hello-microverse
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Awesome books ](#-awesome-books-)
+  - [🛠️ Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Hello-microverse <a name="about-project"></a>
+
+**Hello-microverse**, in this project, i setup only Hello-microverse.
+
+## 🛠️ Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+HTML | CSS 
+
+### Key Features <a name="key-features"></a>
+
+> A simple HTML file that says Hello Microverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [LIVE DEMO UNAVAILABLE]
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+A GitHub | A code editor | A web browser
+
+### Install
+
+Install all project dependencies by running the command below
+
+  `$ npm install`
+
+And run `npm start` to launch the project.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  `git clone https://github.com/hasnatali1947/Hello-microverse`
+
+### Usage
```

Comment:
- [ ] [OPTIONAL] I suggest that you ensure that the correct project setup is used. Depending on the stack used in the project, the setup has always to be customized ☺️ .
![setup](https://user-images.githubusercontent.com/99394374/219031314-0fcfc48f-04d7-4efc-a117-6c09cee6574e.png)